modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/VersionedRowStore.java

```diff
@@ -69,6 +72,22 @@ public class VersionedRowStore {
     public VersionedRowStore(@NotNull MvPartitionStorage storage, @NotNull TxManager txManager) {
         this.storage = Objects.requireNonNull(storage);
         this.txManager = Objects.requireNonNull(txManager);
+
+        Set<RowId> ids = new HashSet<>();
+
+        storage.forEach((rowId, binaryRow) -> {
+            if (ids.add(rowId)) {
+                primaryIndex.put(binaryRow.keySlice(), rowId);
+            }
+        });
+    }
+
+    public void appliedIndex(long appliedIndex) {
```

Comment:
Please add javadoc

Reply:
This java doc is not clear, lets use definition similar to raft paper definition: `lastApplied` -- index of highest log entry applied to state machine (page 4 https://raft.github.io/raft.pdf)